packages/web3-eth-ens/test/fixtures/ens/bytecode/NameWrapperBytecode.ts

```diff
@@ -15,6 +15,6 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-// https://github.com/ensdomains/ens-contracts/blob/master/contracts/resolvers/PublicResolver.sol
+// https://github.com/ensdomains/ens-contracts/blob/1b6b4b59003cceb045634d84ab63ae9bc2c0f8c2/contracts/resolvers/mocks/DummyNameWrapper.sol
 export const NameWrapperBytecode =
```

Comment:
Maybe we need to make a copy of the contract sol. Because it is not our repo. And they can change the solidity code, and bytecode will not be relevant. @jdevcs @nikoulai

Reply:
I think we cover this case with the change suggested by Muhammad (exact commit URL). Adding the contract along with the other contracts in our repo would break their compilation as it has a lot of dependencies and cannot be compiled by our procedure atm. Also, we would have to add all the dependencies, too.

Reply:
Ideally we should have contract code, but current approach (link to contract code specific commit) is also fine for now.